webpack/assets/javascripts/react_app/components/SearchButton/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Button, Col, Icon } from 'patternfly-react';
+
+const SearchButton = props => (
```

Comment:
Shouldn't it suppose to be AutoCompleteSearchButton?
* mostly because `className="autocomplete-search-btn-text"`